twitter_config_sample.py

```diff
@@ -0,0 +1,12 @@
+TWITER_ACCESS_TOKEN="PLACEHOLDER"
+TWITER_ACCESS_TOKEN_ALT="PLACEHOLDER"
```

Comment:
I did notice this typo when I was in here but for the sake of not breaking everyones local developer environments and going down a rabbit hole, I left it as is.